packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -45,3 +46,22 @@ agops perf satisfaction --from "$GOV1ADDR" --executeOffer "$OFFER" --keyring-bac
 test_val $(agoric follow -l -F :published.vaultFactory.managers.manager0.vaults.vault3 -o jsonlines | jq -r '.vaultState') "closed" "vault3 is closed"
 test_val $(agoric follow -l -F :published.vaultFactory.managers.manager0.vaults.vault3 -o jsonlines | jq -r '.locked.value') "0" "vault3 contains no collateral"
 test_val $(agoric follow -l -F :published.vaultFactory.managers.manager0.vaults.vault3 -o jsonlines | jq -r '.debtSnapshot.debt.value') "0" "vault3 has no debt"
+
+upgrade11=$SDK/upgrade-test-scripts/agoric-upgrade-11
+cd $upgrade11
+
+## build proposal and install bundles
+./tools/mint-ist.sh
+./wallet-all-ertp/wf-install-bundles.sh
+
+## upgrade wallet factory
+./wallet-all-ertp/wf-propose.sh
+
+## start game1
+./wallet-all-ertp/wf-game-propose.sh
+
+# Pay 0.25IST join the game and get some places
+node ./wallet-all-ertp/gen-game-offer.mjs Shire Mordor >/tmp/,join.json
+agops perf satisfaction --from $GOV1ADDR --executeOffer /tmp/,join.json --keyring-backend=test
+
+cd $SDK
```

Comment:
is this necessary?

Reply:
no.

it was when the next step was something to do with upgrading zoe.

I'm not sure why I left it in. But doesn't seem worth another ci spin.